data/systems/custom/24_gliese_251.json

```diff
@@ -0,0 +1,497 @@
+			"rotationPhase": "f0/0",
+			"seed": 436556702,
+			"semiMajorAxis": "f0/183836179",
+			"type": "PLANET_TERRESTRIAL",
+			"volatileIces": "f0/0",
+			"volatileLiquid": "f0/0",
+			"volcanicity": "f0/601537672"
+		},
+		{
+			"agricultural": "f0/0",
+			"argOfPeriapsis": "f4294967294/452359824",
+			"aspectRatio": "f1/0",
+			"atmosColor": [
+				0,
+				0,
+				0,
+				0
+			],
+			"atmosDensity": "f0/0",
+			"atmosOxidizing": "f0/3461518337",
+			"averageTemp": 68,
+			"axialTilt": "f0/670329596",
+			"eccentricity": "f0/19368155",
+			"hasRings": false,
+			"humanActivity": "f0/0",
+			"inclination": "f0/737799333",
+			"life": "f0/0",
+			"mass": "f0/297599161",
+			"metallicity": "f0/865149201",
+			"name": "HD 265866 g 2",
+			"orbitalOffset": "f3/2721358045",
+			"orbitalPhase": "f2/3129489614",
+			"parent": 8,
+			"population": "f0/0",
+			"radius": "f0/1737261464",
+			"rotationPeriod": "f4/1789569706",
+			"rotationPhase": "f0/0",
+			"seed": 2620455587,
+			"semiMajorAxis": "f0/239048258",
+			"type": "PLANET_TERRESTRIAL",
+			"volatileIces": "f0/0",
+			"volatileLiquid": "f0/0",
+			"volcanicity": "f0/283909523"
+		},
+		{
+			"agricultural": "f0/0",
+			"argOfPeriapsis": "f4294967295/205910364",
+			"aspectRatio": "f1/0",
+			"atmosColor": [
+				0,
+				0,
+				0,
+				0
+			],
+			"atmosDensity": "f0/0",
+			"atmosOxidizing": "f0/2591924715",
+			"averageTemp": 68,
+			"axialTilt": "f0/365873094",
+			"eccentricity": "f0/11044379",
+			"hasRings": false,
+			"humanActivity": "f0/0",
+			"inclination": "f0/1045546107",
+			"life": "f0/0",
+			"mass": "f0/46732263",
+			"metallicity": "f0/1563299342",
+			"name": "HD 265866 g 3",
+			"orbitalOffset": "f5/4001349339",
+			"orbitalPhase": "f5/2106259025",
+			"parent": 8,
+			"population": "f0/0",
+			"radius": "f0/949836245",
+			"rotationPeriod": "f5/3579139413",
+			"rotationPhase": "f0/0",
+			"seed": 3936335240,
+			"semiMajorAxis": "f0/304167122",
+			"type": "PLANET_TERRESTRIAL",
+			"volatileIces": "f0/0",
+			"volatileLiquid": "f0/0",
+			"volcanicity": "f0/27021706"
+		}
+	],
+	"comment": "  j2000: \"6h54m47.48s/+33°15'56.1\\\"\",\n        dist: 18.2   https://en.wikipedia.org/wiki/Gliese_251\nhttps://www.stellarcatalog.com/exoplanet.php?planetID=100056\nlocal_stars:     {\"name\": \"HD 265866\", \"stars\": [\"STAR_K\"], \"sector\": [1, 0, 1], \"pos\": [6.608, 3.552, 1.864]},\nand :     {\"name\": \"Gliese 251\", \"stars\": [\"STAR_M\"], \"sector\": [1, 0, 1], \"pos\": [7.256, 3.72, 2.312]},\n",
+	"explored": true,
+	"govType": "NONE",
+	"lawlessness": "f0/4284095261",
+	"longDesc": "Gliese 251 is a red dwarf with a spectral type of M3V[3] with an effective temperature of about 3300 K.[3] Its mass has been measured to be around 0.36 solar masses[6] and its radius is about 36% solar radii.[6] Its metallicity is likely slightly less than that of the Sun.[7] Observations at infrared wavelengths rule out the presence of a circumstellar disk around it.",
```

Comment:
Please remove the footnote annotations from this description.

Reply:
Fixed